client/components/Register.jsx

```diff
@@ -37,35 +31,63 @@ function Register() {
   }
 
   const handleSubmit = (evt) => {
+    const gravatar = `https://www.gravatar.com/avatar/${md5(
+      user.email
+    )}?d=retro&f=y`
+
     evt.preventDefault()
```

Comment:
move this up, you want to make sure this is the first line in the event as depending on your network the default could still happen since you have something above it